src/graphql-server/schema.graphql

```diff
@@ -35,6 +35,12 @@ type HostPricing {
 type HolofuelUser {
   id: ID
   nickname: String
+  notFound: Boolean
+}
+
+type ListOrigins {
```

Comment:
This type is not used anywhere, so should be removed.  

(if we were keeping it we'd want to name it a noun not a verb, as it's a type. Probably `OriginList`)

Reply:
This was meant to be removed when it was from the interface and resolvers. Good eye.